web-app/src/lib/db/dto/AgentDTO.ts

```diff
@@ -19,42 +19,62 @@ type AgentWithRelations = Prisma.AgentGetPayload<{
   };
 }>;
 
-export class UnifiedAgent {
-  readonly ranking: bigint;
+export interface Pricing {
+  id: string;
```

Comment:
Lets set all of them to readonly too?

Reply:
3e6b60aebdcc7b6a7afccadd5193296b05f7ff52